index.html

```diff
@@ -146,6 +150,7 @@ <h4 style="color:#b4b4b4; margin:0px;" id='suggestions'><span style="font-size:
               University = this.id;
               document.getElementById("ranklist").innerHTML = "";
               container = [];
```

Comment:
I just mean to say, we don't need to `container = [];` anymore with this new logic.